src/routes/onboarding/glucose-notification-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { Form } from "../../components";
+
+interface Props {
+  index?: number;
+  onDelete?: (index: number) => void;
+  elementId?: number;
+}
+
+export const GlucoseNotificationList: React.FC<Props> = ({ index }) => {
+  return (
+    <div className="flex flex-col max-w-screen-md bg-lightPink relative overflow-hidden m-auto pb-80 h-screen">
```

Comment:
After adding a few reminders they go off screen and you can't scroll down. If you add min-h-fit that should make sure the div is always bigger than the form and fix that problem. Also I don't think you need all that top and bottom padding.

Reply:
Fixed the scrolling issue and removed the padding - the outermost div is really only for conceptualizing the component at the onboarding steps and may be removed anyways depending on how the onboarding wizard is implemented.